### ISSUE TITLE
Website i18n

### ISSUE THREAD

**Issue by ricardobrg** [flutter/website#4760]
I am translating the docs and it gets really hard to keep up with the updates.

Are there any plans on website internationalization? I would like to work on it.

**Comment by sfshaza2** [comment]
No, I'm afraid not. And, I agree, info is being updated at an amazing rate. I am sorry about that.

**Comment by ricardobrg** [comment]
Don't be sorry, it's amazing that it is being updated so much. I will work in a i18n version so it will be easier to mantain. Thanks
